Core/AppRuntime/Source/AppRuntimeChakra.cpp

```diff
@@ -17,8 +18,14 @@ namespace Babylon
         }
     }
 
-    void RuntimeImpl::BaseThreadProcedure()
+    void AppRuntime::CreateEnvironmentAndRun()
     {
+#if WINAPI_FAMILY == WINAPI_FAMILY_DESKTOP_APP
+        HRESULT hr = CoInitializeEx(nullptr, COINIT_APARTMENTTHREADED);
+        assert(SUCCEEDED(hr));
+        auto coInitScopeGuard = gsl::finally([] { CoUninitialize(); });
+#endif
+
```

Comment:
This seems like it should be outside of this class and in AppRuntimeWin32 or something.